### DISCUSSION TITLE
Desert & Lava Ground Type

### DISCUSSION THREAD

**Discussion by ScorchedSky** [Ceddini/CoreKeeperMapTool#24] [Feature Requests]
Hey there, I totally forgot I had an account here but now that I know I have one I can make this request! Your map tool has helped me out so much in my game by helping me narrow down where to look for certain places or bosses. Anyways, I've been able to highlight just the sea water and city grounds to search the Sunken Sea biome for bow parts and I'd like to do the same in the Desert of Beginnings so if it's not too much trouble would you mind adding lava and desert sand ground to the list of options? Please and thank you!

**Comment by Ceddini** [comment]
Will rework all the colors that can be highlighted for the next Core Keeper update (May 10th). So should be released on the 11th or 12th of May.
Didn't really have the time to extract all the color values due to uni, but I will take some time off from my bachelor's thesis to finally add the most important ones. 

Note: You can still highlight a single color by using the color picker. If I remember correctly, it's only available in chrome though

**Comment by Ceddini** [comment]
Added + the new update ground + walls
